### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#55]
Zeus version:
`1.0.32`

Error info:
```Traceback (most recent call last):
  File "zeus.py", line 522, in <module>
    agent=agent_to_use
  File "/home/anonymous/Desktop/Pentest/Tools/Zeus-Scanner/var/google_search/search.py", line 283, in parse_search_results
    url = extract_webcache_url(url)
  File "/home/anonymous/Desktop/Pentest/Tools/Zeus-Scanner/var/google_search/search.py", line 75, in extract_webcache_url
    found = "http:" + to_extract.split(splitter[1])[3]
IndexError: list index out of range
````

Running details:
`Linux-4.10.0-35-generic-x86_64-with-Ubuntu-17.04-zesty`

Commands used:
`zeus.py -l dork.txt`

Log file info:
```2017-10-09 22:31:43,297;zeus-log;INFO;log file being saved to '/home/anonymous/Desktop/Pentest/Tools/Zeus-Scanner/log/zeus-log-12.log'...
2017-10-09 22:31:43,297;zeus-log;INFO;using default search engine...
2017-10-09 22:31:43,298;zeus-log;INFO;starting dork scan with query 'inurl:/general.php?*id=*'...
2017-10-09 22:31:43,298;zeus-log;INFO;attempting to gather query URL...
2017-10-09 22:31:43,485;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-09 22:31:43,486;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-09 22:31:47,107;zeus-log;INFO;browser will open shortly...
2017-10-09 22:31:49,144;zeus-log;INFO;searching 'http://google.com' using query 'inurl:/general.php?*id=*'...
2017-10-09 22:31:52,275;zeus-log;INFO;closing the browser and continuing process..
2017-10-09 22:31:52,313;zeus-log;INFO;URL successfully gathered, searching for GET parameters...
2017-10-09 22:31:52,313;zeus-log;INFO;no proxy configuration detected...
2017-10-09 22:31:52,842;zeus-log;INFO;adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.32)::Python->v2.7'...
2017-10-09 22:31:52,847;zeus-log;INFO;received webcache URL, extracting URL from webcache...
2017-10-09 22:31:52,848;zeus-log;ERROR;[7;31;31mran into exception 'list index out of range'...
Traceback (most recent call last):
  File "zeus.py", line 522, in <module>
    agent=agent_to_use
  File "/home/anonymous/Desktop/Pentest/Tools/Zeus-Scanner/var/google_search/search.py", line 283, in parse_search_results
    url = extract_webcache_url(url)
  File "/home/anonymous/Desktop/Pentest/Tools/Zeus-Scanner/var/google_search/search.py", line 75, in extract_webcache_url
    found = "http:" + to_extract.split(splitter[1])[3]
IndexError: list index out of range

2017-10-09 22:32:29,237;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-09 22:32:29,238;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-09 22:32:29,240;zeus-log;INFO;[32mextracting traceback from log file...[0m
```

**Comment by Ekultek** [comment]
So this happens when the webcache URL does not match correctly, working on it

**Comment by Ekultek** [comment]
```
baal@baal-Aspire-5733Z:~/bin/python/zeus-scanner$ sudo python zeus.py -d inurl:/general.php?*id=* --verbose
[16:07:41 DEBUG] checking if the application has been run before...
[16:07:41 DEBUG] verifying operating system...
[16:07:41 DEBUG] already ran, skipping...

    __          __________                             __   
   / /          \____    /____  __ __  ______          \ \  
  / /    ______   /     // __ \|  |  \/  ___/  ______   \ \ 
  \ \   /_____/  /     /\  ___/|  |  /\___ \  /_____/   / / 
   \_\          /_______ \___  >____//____  >          /_/  
                       \/   \/           \/  v1.0.32(dev)
	https://github.com/ekultek/zeus-scanner.git
		Advanced Dork Searching...


[*] starting up at 16:07:41..


[16:07:41 DEBUG] running with options '{'runInVerbose': True, 'dorkToUse': 'inurl:/general.php?*id=*'}'...
[16:07:41 INFO] log file being saved to '/home/baal/bin/python/zeus-scanner/log/zeus-log-63.log'...
[16:07:41 DEBUG] using default search engine (Google)...
[16:07:41 INFO] using default search engine...
[16:07:41 INFO] starting dork scan with query 'inurl:/general.php?*id=*'...
[16:07:41 DEBUG] checking for user-agent and proxy configuration...
[16:07:41 INFO] attempting to gather query URL...
[16:07:41 DEBUG] setting up the virtual display to hide the browser...
[16:07:41 INFO] firefox browser display will be hidden while it performs the query...
[16:07:41 WARNING] your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
[16:07:41 DEBUG] running selenium-webdriver and launching browser...
[16:07:41 DEBUG] adjusting selenium-webdriver user-agent to 'Zeus-Scanner(v1.0.32)::Python->v2.7'...
[16:07:51 INFO] browser will open shortly...
[16:07:53 DEBUG] searching search engine for the 'q' element (search button)...
[16:07:53 INFO] searching 'http://google.com' using query 'inurl:/general.php?*id=*'...
[16:07:57 DEBUG] obtaining URL from selenium...
[16:07:57 DEBUG] found current URL from selenium browser...
[16:07:57 INFO] closing the browser and continuing process..
[16:07:57 INFO] URL successfully gathered, searching for GET parameters...
[16:07:57 INFO] no proxy configuration detected...
[16:07:58 INFO] adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.32)::Python->v2.7'...
[16:07:58 DEBUG] found 'http://search.clearch.org/?a=images&q=Inurl+General.php%3F'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'http://search.clearch.org/?a=images&q=Inurl'...
[16:07:58 DEBUG] found 'http://search.clearch.org/?a=images&q=Inurl+General.php%3F%2Aid%3D%2A'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'http://search.clearch.org/?a=images&q=Inurl'...
[16:07:58 DEBUG] found 'http://www.reelshotwaterfowl.com/general.php?id=7'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'http://www.reelshotwaterfowl.com/general.php?id=7'...
[16:07:58 DEBUG] found 'http://siam-indo.com/general.php?id=8'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'http://siam-indo.com/general.php?id=8'...
[16:07:58 DEBUG] found 'http://www.s8wministries.org/general.php?id=15'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'http://www.s8wministries.org/general.php?id=15'...
[16:07:58 DEBUG] found 'http://www.arabculturefund.org/grants/general.php?id=10s'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'http://www.arabculturefund.org/grants/general.php?id=10s'...
[16:07:58 DEBUG] found 'http://www.dan-gulf.com/general.php?id=20'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'http://www.dan-gulf.com/general.php?id=20'...
[16:07:58 DEBUG] found 'http://www.ummqarn.com/stallions/general.php?ID=136'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'http://www.ummqarn.com/stallions/general.php?ID=136'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'https://github.com/ULCC/theme_newham/blob/master/layout/general.php'...
[16:07:58 INFO] received webcache URL, extracting URL from webcache...
[16:07:58 DEBUG] found 'https://github.com/WordPress/WordPress/blob/master/wp-admin/options-general.php'...
[16:07:58 INFO] found a total of 11 URL's with a GET parameter...
[16:07:58 INFO] successfully wrote found items to '/home/baal/bin/python/zeus-scanner/log/url-log/url-log-8.log'...


[*] shutting down at 16:07:58..


baal@baal-Aspire-5733Z:~/bin/python/zeus-scanner$ 
```

**Comment by Ekultek** [comment]
fixed via e43ee8a
